### PR TITLE
Review and update intro and quickstart docs

### PR DESCRIPTION
**Pull request by dbtucker**
Some simple cleanup of the documented details on our docker images and the commands to deploy them

### CODE REVIEW

File: docs/intro.rst [hunk 1]
```diff
@@ -76,27 +78,31 @@ Important Notes/Caveats
...
 	* Add the connector jars via volumes.  If you don't want to create a new Docker image, please see our documentation on `Configuring Kafka Connect with External Jars <operations/external-volumes.html>`_ to configure the `cp-kafka-connect` container with external jars.
 
+6. Included Java
+
+    The Confluent Docker images are tested and shipped with `Azul Zulu OpenJDK <https://www.azul.com/products/zulu/>`_.  Other JDK's (including Oracle Java) are supported, but you must extend the images yourself to implement that change. 
+
 7. Untested Features
 
-	The following features/conditions are not currently tested:
+	The following features/enfironments are not currently tested:
```
Comment (samjhecht) [inline docs/intro.rst]: spelling of "environments"

File: docs/intro.rst [hunk 2]
```diff
@@ -76,27 +78,31 @@ Important Notes/Caveats
...
 
-	* Build a new Docker image that has connector installed. You can follow the examples found in our documentation on `Extending Images <development.html#extending-the-docker-images>`_. You will need to make sure that the connector jars are on the classpath.
+	* Build a new Docker image that has the connector installed. You can follow the examples found in `Extending Images <development.html#extending-the-docker-images>`_. You will need to make sure that the connector jars are on the CLASSPATH for the Connect service (the default location of /usr/share/java/kafka-connect-* is the recommended location).
 	* Add the connector jars via volumes.  If you don't want to create a new Docker image, please see our documentation on `Configuring Kafka Connect with External Jars <operations/external-volumes.html>`_ to configure the `cp-kafka-connect` container with external jars.
 
+6. Included Java
+
+    The Confluent Docker images are tested and shipped with `Azul Zulu OpenJDK <https://www.azul.com/products/zulu/>`_.  Other JDK's (including Oracle Java) are supported, but you must extend the images yourself to implement that change.
```
Comment (samjhecht) [inline docs/intro.rst]: are all other JDK options supported?  This makes it sounds like others are being tested.

File: docs/quickstart.rst [hunk 3]
```diff
@@ -3,11 +3,9 @@
 Quickstart
 ==========
 
-In this section, we provide a simple guide for running a Kafka cluster along with all of the other Confluent Platform components.  By the end of this quickstart, you will have successfully installed and run a simple deployment including each component with Docker.
+This section provides basic guide for deploying a Kafka cluster along with all Confluent Platform components in your Docker environment.  By the end of this quickstart, you will have a functional Confluent deployment against which you can run any number of applications.
```
Comment (samjhecht) [inline docs/quickstart.rst]: provides "a" basic guide...

File: docs/quickstart.rst [hunk 4]
```diff
@@ -22,28 +20,30 @@ Running Confluent Platform on Docker
...
 
     eval $(docker-machine env confluent)
 
+All of the subsequent commands should be run from that terminal window to ensure proper access to the running Docker host.  To execute Docker commands from a new terminal window, simply execute the ``eval $(docker-machine env confluent)`` first.
+
 Zookeeper
 +++++++++++++++++
 
-Start Zookeeper. You'll need to keep this service running throughout, so if you will be running things in the foreground, you'll need to have it in a dedicated terminal window.
+Start Zookeeper. You'll need to keep this service running throughout, so if used a dedicated terminal window if you plan to launch the image in the foreground.
```
Comment (samjhecht) [inline docs/quickstart.rst]: "so use a dedicated terminal window..."

File: docs/quickstart.rst [hunk 5]
```diff
@@ -69,6 +69,13 @@ Start Zookeeper. You'll need to keep this service running throughout, so if you
 
     [2016-07-24 05:15:35,453] INFO binding to port 0.0.0.0/0.0.0.0:32181 (org.apache.zookeeper.server.NIOServerCnxnFactory)
 
+  Note that the message shows the Zookeeper service listening at the port we passed in as ``ZOOKEEPER_CLIENT_PORT`` above.
+
+  If the service is not running, the log messages should provide details to help you identify the problem.   Some common errors include:
+
+		* Network port already in use .   In that case, you'll see a message indicating that the Zookeeper service could not bind to the selcted port.  Simply change to an open port or identify (and stop) the Docker container that has a service using that port.
```
Comment (samjhecht) [inline docs/quickstart.rst]: extra space at the end of "Network port already in use ."

File: docs/quickstart.rst [hunk 6]
```diff
@@ -244,13 +252,13 @@ Consume data via the REST Proxy.
       -e KAFKA_REST_HOST_NAME=localhost \
       confluentinc/cp-kafka-rest:3.1.1
 
-  For the next two steps, we're going to use CURL commands to talk to the REST Proxy. For the sake of simplicity, the Schema Registry and REST Proxy containers on same host with the REST Proxy listening at http://localhost:8082.
+  For the next two steps, we're going to use CURL commands to talk to the REST Proxy. Our deployment steps thus far have ensured that both the REST Proxy container and the Schema Registry container are accessible directly through network ports on our local host.  The REST Proxy service is listening at http://localhost:8082  As above, we'll launch a new Docker container from which to execute our commands :
```
Comment (samjhecht) [inline docs/quickstart.rst]: Extra space at the end before the colon.

File: docs/quickstart.rst [hunk 7]
```diff
@@ -419,16 +446,18 @@ consumption and expected consumption is greater than 1000 messages:
...
 
 Set the trigger metric to be "Consumption difference" where the
 condition is "Greater than" 1000 messages. The buffer time (in seconds) is the
 wall clock time we will wait before firing the trigger to make sure the trigger
 condition is not too transient.
 
-After saving the trigger, Control Center will now prompt us to associate an action that will fire when
-our newly created trigger occurs. For now, the only action is send an email.
+After saving the trigger, Control Center will now prompt us to associate an action that will execute when
+our newly created trigger fires. For now, the only action is send an email.
```
Comment (samjhecht) [inline docs/quickstart.rst]: is "to" send an email...

File: docs/intro.rst (hunk 2 above)
Comment (dbtucker) [inline docs/intro.rst]: We support other JDK's explicitly elsewhere in our docs.   It seems risky to say that the Docker images only support a JVM that we don't mention anywhere else.

**Pull request merged**